src/plugins/kibana/server/lib/ingest/processors/convert/__tests__/converter.js

```diff
@@ -114,36 +104,36 @@ describe('ingest', () => {
             expect(_.isEqual(actual, expected)).to.be.ok();
           });
 
-          it('should map [auto] type correctly', () => {
-            source.type = 'auto';
-            expected.type = 'auto';
-
-            const actual = esToKibana(source);
-            expect(_.isEqual(actual, expected)).to.be.ok();
-          });
+          it('should map type [double], [float], [int], [long] to [number]', () => {
+            source.type = 'double';
+            expected.type = 'number';
+            expect(_.isEqual(esToKibana(source), expected)).to.be.ok();
 
-          it('should map [float] type correctly', () => {
             source.type = 'float';
             expected.type = 'number';
+            expect(_.isEqual(esToKibana(source), expected)).to.be.ok();
 
-            const actual = esToKibana(source);
-            expect(_.isEqual(actual, expected)).to.be.ok();
+            source.type = 'int';
```

Comment:
This should actually be `integer`, not `int`. There's also a `short` numeric type.